css/main.css

```diff
@@ -1,9 +1,127 @@
 
-
 /* ==========================================================================
    Author's custom styles
    ========================================================================== */
-
+   .fa {
+    padding: 20px;
+    font-size: 30px;
+    width: 50px;
```

Comment:
Hi Tyrone - I copied these files to see the results, and I think a width of 75px works better here.  I saw that the boxes were a bit squashed together horizontally, and this width helps space them out nicely.  If you like the change, I would also recommend a "margin-top: 5px;" to fix the spacing vertically when the icons spill onto the next line.